modules/social_features/social_comment/social_comment.tokens.inc

```diff
@@ -83,17 +83,24 @@ function social_comment_tokens($type, $tokens, array $data, array $options, Bubb
             if (isset($message->field_message_related_object)) {
               $target_type = $message->field_message_related_object->target_type;
               $target_id = $message->field_message_related_object->target_id;
+              /** @var \Drupal\comment\Entity\Comment $comment */
               $comment = \Drupal::entityTypeManager()
                 ->getStorage($target_type)
                 ->load($target_id);
 
               if (!empty($comment)) {
                 if ($comment->getEntityTypeId() == 'comment') {
                   if (!empty($comment->getCommentedEntity())) {
+                    /** @var \Drupal\Core\Entity\Entity $node */
                     $node = $comment->getCommentedEntity();
```

Comment:
We should probably rename the occurrences of `$node` to `$entity` since this piece of code also works for the "Post" entity which is not a node.

We should ensure that the naming doesn't bite us in the future.